.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+Description of work.
+
+**To test:** Instructions for testing.
+
+Fixes #xxxx.
+
+Either link to [Release notes](http://www.mantidproject.org/Release_Notes_3.7) change-set or state "Does not need to be in the release notes."
```

Comment:
Would it be easier to have the release notes in the source and build them with sphinx? Then we wouldn't need to link to anything, everything would be right in the PR.